wdl2cwl/main.py

```diff
@@ -650,10 +650,15 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:
             only_arg = arguments[0]
             return self.get_expr(only_arg)
         elif function_name == "select_first":
-            array_obj = arguments[0]
-            array_items = [str(self.get_expr(item)) for item in array_obj.items]  # type: ignore
+            array_obj = cast(WDL.Expr.Array, arguments[0])
+            array_items = [str(self.get_expr(item)) for item in array_obj.items]
```

Comment:
Fixed the mypy warning. Also, the arrow function caused a warning when validating the workflow, as that's not part of EC5